index.html

```diff
@@ -4003,15 +4150,35 @@ <h3>Algorithm</h3>
           value matches the <a>type mapping</a> of <var>active property</var>,
           return the value associated with the <code>@value</code> <a>entry</a>
           of <var>value</var>.</li>
-        <li>Otherwise, if <var>value</var> has an <code>@language</code> <a>entry</a> whose
-          value matches the <a>language mapping</a> of
-          <var>active property</var>, return the value associated with the
+        <li class="changed">Otherwise, if <var>value</var> has both `@language` and `@direction <a>entries</a>,
+          and the `@language` <a>entry</a> matches either the <a>language mapping</a> of
+          <var>active property</var> or the <a>default language</a> of <var>active context</var>
```

Comment:
shouldn't that be

* it matches the language mapping of the active property OR
* **the language mapping of the active property is null** and it matches the default language of the active context
?

Same for the the direction?

Reply:
I think this was resolved in subsequent changes, can you verify?